### PR TITLE
Remove gone urltools

### LINKED ISSUE #61: poetry install fails, urltools v0.3.2 is gone including the original repo
```
❯ poetry install
Installing dependencies from lock file


Package operations: 1 install, 0 updates, 0 removals

  - Installing urltools (0.3.2)

[EnvCommandError]
Command ['/home/seb/.cache/pypoetry/virtualenvs/spkrepo-4s3rbOEh-py3.9/bin/pip', 'install', '--no-deps', 'urltools==0.3.2'] errored with the following return code 1, and output: 
ERROR: Could not find a version that satisfies the requirement urltools==0.3.2 (from versions: 0.4.0)
ERROR: No matching distribution found for urltools==0.3.2
WARNING: You are using pip version 20.2.3; however, version 21.0 is available.
You should consider upgrading via the '/home/seb/.cache/pypoetry/virtualenvs/spkrepo-4s3rbOEh-py3.9/bin/python -m pip install --upgrade pip' command.
```

### PR DESCRIPTION
**Pull request by publicarray**
Removed urltools and disabled the offending test case
updated dependencies
updated readme﻿

### CODE REVIEW

**Comment by publicarray** [comment]
Build fails because of this: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

File: spkrepo/tests/test_nas.py [hunk 1]
```diff
@@ -32,12 +30,11 @@ def assertCatalogEntry(self, entry, build, data=None, link_params=None):
...
-            compare(
-                entry["link"],
-                url_for("nas.data", path=build.path, **link_params, _external=True),
-            )
-        )
+        # self.assertTrue(
+        #     compare(
+        #         entry["link"],
+        #         url_for("nas.data", path=build.path, **link_params, _external=True),
+        #     )
```
Comment (Diaoul) [inline spkrepo/tests/test_nas.py:37]: Please delete rather than comment

**Review by Diaoul — CHANGES_REQUESTED** [review]

**Comment by Diaoul** [comment]
Thanks for the PR! LGTM besides my other comment.

**Review by Diaoul — APPROVED** [review]

**Comment by publicarray** [comment]
Thanks @Diaoul!

**Pull request merged**